pkg/http/responses_fasthttp.go

```diff
@@ -78,6 +80,14 @@ func universalFastHTTPErrorResponder(reqCtx *fasthttp.RequestCtx, err error) {
 		return
 	}
 
+	// Check if it's a kitErrors.Error object
+	var kitErr kiterrors.Error
+	ok = errors.As(err, &kitErr)
```

Comment:
```suggestion
	standardizedErr, ok := kitErrors.FromError(err)
```

I think we can use the `FromError()` here and then pass `standardizedErr` to the `fasthttpRespond`

Reply:
Ah, thanks! I knew there were more places where we were doing this, but couldn't find it. Fixed.